src/utils/utils.js

```diff
@@ -1,12 +1,13 @@
+import { getNetworkId } from '@ensdomains/ui/src/web3'
```

Comment:
I assume this is for the reducing the bundle size?

If so can you write some docs to explain what developers need to do to adhere to this standard.